CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+
+## Solve an Issue
+
+Scan through our [existing issues](https://github.com/cancerDHC/ccdhmodel/issues) to find one that interests you.
+
+* The issues can be self-assigned and/or assigned by any members of the team. Please do not start work on an assigned issue without checking with the assignee, who might be already working on the issue, and please unassign yourself if you can no longer work on that issue.
+* The issues we should tackle first are listed on 'standing reference' section of the Tools team meeting notes.
+* Issues we are currently working on are grouped using [Milestones](https://github.com/cancerDHC/ccdhmodel/milestones). Milestones group thematically related issues that are due to be completed by a particular date.
+
+### Create a Branch
+
+We follow the [gitflow](https://nvie.com/posts/a-successful-git-branching-model/) workflow for branching practice and naming convention.
+(Discussions on branching model and branching practice are ongoing.)
+
+* The repo holds two main branches with an infinite lifetime: `main` and `develop`
+* The supporting branches we may use are:
+  * Feature branches
+  * Release branches
+  * Hotfix branches
+* Naming convention of the branches
+  * Feature branches: You can name your branch anything except `main` and `develop`. You can add a short descriptor of the task, and/or an issue ID to the name of the branch.
+  * Release branches: release-*
+  * Hotfix branches: hotfix-*
+* It is recommended to use the [GitFlow Git extension](https://github.com/nvie/gitflow) to implement the gitflow branching model locally.
+
+### Pull Request
+
+When you're finished with the changes on your code, create a [pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests), also known as a PR.
+
+* PRs for feature branches should always merge into `develop`, while PRs for release and hotfix branches should always merge into `main`, unless there are good reasons to merge them into another branch.
+* Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one. A good way to do this is by writing `closes #[issue number]` somewhere in the PR description.
+* Major changes, especially new features, should be documented in the CHANGELOG file in the PR so we can keep track of which version each change was incorporated into.
+
+### Code Review
+
+A pull request must be reviewed by **at least one other project member** before it is merged.
+
+> A CCDH engineer must review and accept your pull request. A code review (which happens with both the contributor and the reviewer present) is required for contributing.
```

Comment:
I'm not sure if this is needed any more? I think the previous sentence ("A pull request must be reviewed by **at least one other project member** before it is merged.") should be sufficient.

Reply:
Okay. I added the sentence as @sujaypatil96 suggested. The first sentence would be sufficient, since PR would not get automatically merged anyway. 

Thank you for your feedback, @gaurav!